test/unit/socialServerMock.ts

```diff
@@ -0,0 +1,69 @@
+const jsonServer = require('json-server')
+
+export function mockSocialServer(userId: string, validToken: string, anotherUserId: string, PORT: number) {
+    const server = jsonServer.create()
+
+    // No friends userId
+    server.get(`/v1/friendships/${userId}`, (req, res) => {
+        if (isValidToken(req)) {
+            res.jsonp({
+                friendships: []
+            })
+        } else {
+            res.sendStatus(401)
+        }
```

Comment:
Let's wrap the logic of the `if (isValidToken) { ... } else { res.sendStatus(401) }` as another function. WDYT?